core/modules/replica/HttpRequestsModule.cc

```diff
@@ -0,0 +1,125 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "replica/HttpRequestsModule.h"
+
+// System headers
+#include <stdexcept>
+
+// Qserv headers
+#include "replica/DatabaseServices.h"
+#include "replica/HttpRequestQuery.h"
+#include "replica/ServiceProvider.h"
+
+using namespace std;
+using json = nlohmann::json;
+
+namespace lsst {
+namespace qserv {
+namespace replica {
+
+HttpRequestsModule::Ptr HttpRequestsModule::create(Controller::Ptr const& controller,
+                                                   string const& taskName,
+                                                   unsigned int workerResponseTimeoutSec) {
+    return Ptr(new HttpRequestsModule(
+        controller,
+        taskName,
+        workerResponseTimeoutSec
+    ));
+}
+
+
+HttpRequestsModule::HttpRequestsModule(Controller::Ptr const& controller,
+                                       string const& taskName,
+                                       unsigned int workerResponseTimeoutSec)
+    :   HttpModule(controller,
+                   taskName,
+                   workerResponseTimeoutSec) {
+}
+
+
+void HttpRequestsModule::executeImpl(qhttp::Request::Ptr const& req,
+                                     qhttp::Response::Ptr const& resp,
+                                     string const& subModuleName) {
+
+    if      (subModuleName.empty()) _requests(req, resp);
```

Comment:
mixing with and without brackets in the if/else if/else clause is suspect, IMO.

Reply:
The code is really trivial here. It's meant to dispatch incoming requests to the corresponding sub-modules. The way it's written is meant to make it more readable. I can still fix it to be in full compliance with the Style Guide.

Reply:
For the record. Fixed in this and other modules.